### PR TITLE
update asciidoctor-multipage to fix style generation problem

### PR DESCRIPTION
**Pull request by diguage**

### CODE REVIEW

File: asciidoc-to-html-multipage-example/pom.xml [hunk 1]
```diff
@@ -90,8 +90,7 @@
                         <project-version>${project.version}</project-version>
                         <sourcedir>${project.build.sourceDirectory}</sourcedir>
                         <source-highlighter>rouge</source-highlighter>
-                        <rouge-linenums-mode>table</rouge-linenums-mode>
-                        <rouge-style>monokai</rouge-style>
+                        <rouge-style>github</rouge-style>
```
[the remark this answers is not in the document]
Comment (abelsromero) [reply asciidoc-to-html-multipage-example/pom.xml:93]: Maybe we can merge only the version bump and leave the example as it is with a comment to the PR stating we add custom style as a file in `assets/styles` as a temporal workarround.

[the remark this answers is not in the document]
Comment (mojavelinux) [reply asciidoc-to-html-multipage-example/pom.xml:93]: I'm not sure I understand what's going on here. Rouge supports the monokai style. See https://github.com/rouge-ruby/rouge/tree/master/lib/rouge/themes

[the remark this answers is not in the document]
Comment (diguage) [reply asciidoc-to-html-multipage-example/pom.xml:93]: @mojavelinux This is the replay about the problem: https://github.com/owenh000/asciidoctor-multipage/issues/35#issuecomment-1155985609

[the remark this answers is not in the document]
Comment (diguage) [reply asciidoc-to-html-multipage-example/pom.xml:93]: The PR is OK.

But if change the `rouge-style` to the other options, the highlighter does not work(the linked style and the generated style are not the same files.)

[the remark this answers is not in the document]
Comment (mojavelinux) [reply asciidoc-to-html-multipage-example/pom.xml:93]: Aha, I see. Thanks for the clarification.

[the remark this answers is not in the document]
Comment (abelsromero) [reply asciidoc-to-html-multipage-example/pom.xml:93]: If it's fine for everyone I'll merge.

**Pull request merged**